ethcore/src/executive.rs

```diff
@@ -573,13 +573,16 @@ impl<'a, B: 'a + StateBackend> Executive<'a, B> {
 		vm_tracer.done_subtrace(subvmtracer);
 
 		match res {
-			Ok(ref res) if res.apply_state => tracer.trace_create(
-				trace_info,
-				gas - res.gas_left,
-				trace_output.map(|data| output.as_ref().map(|out| out.to_vec()).unwrap_or(data)),
-				created,
-				subtracer.drain()
-			),
+			Ok(ref res) if res.apply_state => {
+				substate.contracts_created.push(created.clone());
```

Comment:
Is it used only for tests? Might be better to set up a tracer for tests instead.

Reply:
I see it was moved from `externalities`. Seems it makes sense.